src/NHSD.BuyingCatalogue.Solutions.Contracts/Commands/BrowserBased/IUpdateBrowserBasedConnectivityAndResolutionData.cs

```diff
@@ -0,0 +1,11 @@
+namespace NHSD.BuyingCatalogue.Solutions.Contracts.Commands.BrowserBased
+{
+    public interface IUpdateBrowserBasedConnectivityAndResolutionData
+    {
+        string MinimumConnectionSpeed { get; }
+
+        string MinimumDesktopResolution { get; }
+
+        IUpdateBrowserBasedConnectivityAndResolutionData Trim();
```

Comment:
I'm not sure about the trim on this interface. It is meant to be a simple contract representing the data of a command. I would prefer if we created an extension method in the API for this functionality. At the moment the handler now has access to this trim method, which seems like it should only be the responsibility of the "view" (API).